File: RepoTransBench/source_projects/Python/RDFLib_rdflib-jsonld/tests/test_init.py
# Fix import failure by ensuring the package directory is in sys.path
import sys
import os

sys.path.insert(0, os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))

def test_import_rdflib_jsonld():
    import rdflib_jsonld
    assert hasattr(rdflib_jsonld, '__doc__')
    assert hasattr(rdflib_jsonld, '__version__')
    assert hasattr(rdflib_jsonld, '__author__')
    assert hasattr(rdflib_jsonld, '__contact__')
    assert hasattr(rdflib_jsonld, '__docformat__')

File: RepoTransBench/target_projects/Python/Rust/RDFLib_rdflib-jsonld/tests/original/test_init.rs
// There is no import failure in Rust, but we can check all constants are accessible
use rdflib_jsonld::*;

#[test]
fn test_import_rdflib_jsonld() {
    // All symbols from lib.rs must be available
    let _: &str = VERSION;
    let _: &str = AUTHOR;
    let _: &str = CONTACT;
    let _: &str = DOCFORMAT;
    // Doc is checked by doc-test
    assert!(true);
}